hedera-node/test-clients/src/main/java/com/hedera/services/bdd/junit/SubProcessHapiTestNode.java

```diff
@@ -88,6 +88,7 @@ public void start() {
                             javaCmd,
                             "-classpath",
                             classPath,
+                            "-Dfile.encoding=utf-8",
```

Comment:
It appears as if this is not picked up by the built. It is still complaining about the wrong default character set.

Reply:
We need to actually modify the environment, something like this:
```java
            final var builder = new ProcessBuilder();
            final var environment = builder.environment();
            environment.put("LC_ALL", "en@UTF-8");
            builder.command(
                        javaCmd,
                        "-classpath",
                        classPath,
                        "-Dfile.encoding=utf-8",
                        "-Dhedera.workflows.enabled=true",
                        "com.hedera.node.app.ServicesMain",
                        "" + nodeId)
                    .directory(workingDir.toFile())
                    .redirectOutput(stdout.toFile())
                    .redirectError(stderr.toFile());
```